gulpfile.js

```diff
@@ -129,7 +131,21 @@ defineBuildTask('HTMLImports');
 defineBuildTask('ShadowDOM');
 defineBuildTask('MutationObserver');
 
-gulp.task('build', ['webcomponents', 'webcomponents-lite', 'CustomElements', 
+gulp.task('CustomElementsV1', function () {
+  return gulp.src('./src/CustomElements/v1/CustomElements.js', {base: './'})
+      .pipe(closureCompiler({
```

Comment:
Using ES6 modules (or CommonJS) would encourage much more contribution. Developers could import the modules into any runtime or build environment that supports ES6 Modules, and won't be strictly required to use your build system.

Currently, developers are required to use your build flow or make a brand new one in order to create something usable, and it is difficult because they have to make sure to load the files in the correct order. Using a module system solves that problem.

If you switch to a module system, you can still make a global by creating an index file that imports whatever it needs then setting those things onto `window`.

I would highly recommend making the switch.